### ISSUE TITLE
Works only with n_feats=80

### ISSUE THREAD

**Issue by patriotyk** [p0p4k/pflowtts_pytorch#26]
I have tried to change n_feats=100 and train, but looks like there is several places hardcoded for 80 because I got crashes that say it expects size 80.

**Comment by p0p4k** [comment]
Yes, manually replace 80 with 100. There are different modules giving out 80 sized outputs, they can be further edited to change their output sizes, so better to change it to 80 wherever needed. In my another encodec branch, it gives out 128 feats, you can use it for reference.

**Comment by patriotyk** [comment]
Thank you!!!

**Comment by patriotyk** [comment]
It depends on vocoder. So if you have vocoder trained with 100 features, pflow model also should generate mels with 100 features.

**Comment by patriotyk** [comment]
In my case I just wanted to try pflow with pretrained vocos vocoder. I just wanted to compare it with pretrained hifigan. I suppose it should sound better but I don't know for sure.
